zap/src/main/java/org/zaproxy/zap/view/AbstractMultipleOptionsBaseTablePanel.java

```diff
@@ -170,17 +173,43 @@ public void mousePressed(MouseEvent me) {
         }
     }
 
+    protected AbstractMultipleOptionsBaseTablePanel(
+            boolean allowDuplicate, AbstractMultipleOptionsBaseTableModel<E> model) {
+        super(model);
+
+        if (allowDuplicate) {
```

Comment:
Doing this here means the duplicate will be the first button, IMO it should be after the add button.

Reply:
Cool. I'll initialize the add button in this constructor

Reply:
Better if the logic is kept in the same constructor, add a private constructor which is called by both constructors.